### DISCUSSION TITLE
audio gain

### DISCUSSION THREAD

**Discussion by thewayur** [mikebrady/shairport-sync#1651] [Ideas]
I m using android and x.x.x.x app to stream audio to airplay (and it is using classic airplay).
but as per android system, the captured volume is low in compare to USB/headphone. So the received audio is also low   in airplay.

So, a replay gain option would be nice :)
Also it would be more beneficial if there would be anti-clip option for accidental sounds :)

Thanks for this wonderful software, happily using since many years :)

**Comment by mikebrady** [comment]
Hmm, this might be outside the scope of Shairport Sync. The maximum output level from Shairport Sync, by default, is the maximum output level of the output device.
So I wonder, where is the low volume problem coming from? Is it the x.x.x.x app itself. (I'm a bit confused.)

**Reply by thewayur** [reply]
the input volume (from my mobile app) is low, hence overall output volume is low on raspberry,
never mind, i used vlc/pulseaudio for output gain.

problem is solved but output control via shairport would be handy.
thanks. the problem is solved! :)
